code/modules/clothing/modular/modular.dm

```diff
@@ -0,0 +1,217 @@
+/**
+	Modular armor
+*/
+/obj/item/clothing/suit/modular
+	name = "Jaeger XM-02 combat exoskeleton"
+	desc = "Designed to mount a variety of modular armor components and support systems. It comes installed with light-plating and a shoulder lamp. Mount armor pieces to it by clicking on the frame with the components"
+	icon = 'icons/mob/modular/modular_armor.dmi'
+	icon_state = "underarmor_icon"
+	item_state = "underarmor"
+	flags_armor_protection = CHEST|GROIN|ARMS|LEGS
+	/// What is allowed to be equipped in suit storage
+	allowed = list(
+		/obj/item/weapon/gun/,
+		/obj/item/tank/emergency_oxygen,
+		/obj/item/storage/bible,
+		/obj/item/storage/belt/sparepouch,
+		/obj/item/storage/large_holster/machete,
+		/obj/item/weapon/claymore,
+		/obj/item/storage/belt/gun
+	)
+	flags_equip_slot = ITEM_SLOT_OCLOTHING
+	w_class = WEIGHT_CLASS_BULKY
+
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
+	siemens_coefficient = 0.9
+	permeability_coefficient = 1
+	gas_transfer_coefficient = 1
+
+	actions_types = list(/datum/action/item_action/toggle)
+
+	/// An assoc list of stat mods
+	var/list/stat_mod
```

Comment:
Plans for this?

Reply:
It was left over, will remove i wasn't sure how to approach the stat modifiers